src/cli/formatters/__tests__/pretty.test.ts

```diff
@@ -0,0 +1,112 @@
+import { DiagnosticSeverity } from '@stoplight/types';
+import * as chalk from 'chalk';
+import { sortResults } from '../../../utils';
+import { pretty } from '../pretty';
+
+const oas3SchemaErrors = sortResults(require('./__fixtures__/oas3-schema-errors.json'));
+const mixedErrors = sortResults(require('./__fixtures__/mixed-errors.json'));
+
+function setColumnWidth(width: number, func: CallableFunction): void {
+  const og = process.stdout.columns;
+  process.stdout.columns = width;
+  func(width);
+  process.stdout.columns = og;
```

Comment:
```suggestion
  try {
    func(width);
  } finally {
    process.stdout.columns = og;
  }
```
just in case some assertion fails.
It shouldn't be a big deal, but if one makes certain changes and we have other tests that are sensitive to the width, it may potentially cause confusions for some.